product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx

```diff
@@ -195,7 +195,15 @@ An Oracle URL contains the following information:
 
 -   `<service_name>` &mdash; The name of the Oracle service.
 
--   `SRC_DB_USER` &mdash; Specifies the name of a privileged Oracle user. The Oracle user needs DBA privilege to migrate objects from Oracle to Advanced Server. The DBA privilege can be granted to the Oracle user with the Oracle `GRANT DBA TO user` command to ensure all of the desired database objects are migrated.
+-   `SRC_DB_USER` &mdash; Specifies the name of a privileged Oracle user. The Oracle user needs read access to the source database objects you want to migrate. If you want to migrate users/roles and related profiles/privileges, grant the Oracle user SELECT privileges on the following Oracle catalogs:
+    
+    - `DBA_ROLES`
+    - `DBA_USERS`
+    - `DBA_TAB_PRIVS`
+    - `DBA_PROFILES`
+    - `DBA_ROLE_PRIVS`
+    - `ROLE_ROLE_PRIVS`
+    - `DBA_SYS_PRIVS`
 
```

Comment:
In the rendered page (https://deploy-preview-5798--edb-docs-staging.netlify.app/docs/migration_toolkit/latest/06_building_toolkit.properties_file/#defining-an-oracle-url), the spacing between the catalog object and the next connection parameter (`SRC_DB_PASSWORD`) does not look right.

See:
<img width="911" alt="image" src="https://github.com/EnterpriseDB/docs/assets/63003848/f762a46f-4dd0-4933-86ed-cf417c5c9c97">